docs/docs/how-tos/human_in_the_loop/add-human-in-the-loop.md

```diff
@@ -171,6 +172,162 @@ resume_map = {
 parent_graph.invoke(Command(resume=resume_map), config=thread_config)
 ```
 
+## Static interrupts
```

Comment:
IMO it should be much later in the document since breakpoints aren't as much of a focus for HIL.